src/views/GameListView.jsx

```diff
@@ -88,11 +141,26 @@ const cleanWord = function (word) {
   return Diacritics.clean(word).toLowerCase();
 };
 
+const hasIntervalOverlap = function (interval1, interval2) {
+  return interval1[0] <= interval2[1] && interval1[1] >= interval2[0];
+};
+
 const GameListView = () => {
   const { t } = useTranslation();
   const NULL_SEARCH_TERM = "";
+  const MULTI_LANG_KEYWORD = "Multi-lang";
 
-  const [searchTerm, setSearchTerm] = React.useState(NULL_SEARCH_TERM);
+  const [filterCriteria, setFilterCriteria] = React.useState({
+    searchTerm: NULL_SEARCH_TERM,
+    nbOfPlayersMin: 1,
+    nbOfPlayersMax: 9,
+    durationMin: 15,
+    durationMax: 90,
+    language: {
+      fr: true,
+      en: true,
+    },
+  });
```

Comment:
Is it easier to use if filter intervals are already lists ?

```suggestion
    searchTerm: NULL_SEARCH_TERM,
    nbOfPlayers: [1, 9],
    duration: [15, 90],
    languages: ["fr", "en"],
  });
```

Reply:
Not sure. Makes it a bit less readable/understandable as well.

Reply:
You brought good arguments and examples though, I will give it a try.